taipy/gui/gui_actions.py

```diff
@@ -440,3 +440,17 @@ def thread_status(name: str, period_s: float, count: int):
     thread.start()
     if isinstance(period, int) and period >= 500 and callable(user_status_function):
         thread_status(thread.name, period / 1000.0, 0)
+
+
+def get_local_storage(state: State, *keys: str) -> t.Optional[t.Union[str, t.Dict[str, str]]]:
+    """Get local storage value(s).
+    Arguments:
+        state (State^): The current user state as received in any callback.
+        *keys (string): The keys to get from the local storage
+    Returns:
```

Comment:
Line skip.

Explanations are pretty weak. Let's spend some time together on this one.

Reply:
i am open to any proposal to the doc